src/foam/u2/view/TitledArrayView.js

```diff
@@ -0,0 +1,79 @@
+/**
+ * @license
+ * Copyright 2021 The FOAM Authors. All Rights Reserved.
+ * http://www.apache.org/licenses/LICENSE-2.0
+ */
+
+foam.CLASS({
+  package: 'foam.u2.view',
+  name: 'TitledArrayView',
+  extends: 'foam.u2.view.FObjectArrayView',
+
+  documentation: 'View for FObjectArrays that includes a header row with a title and a delete button',
+  imports: ['theme'],
+
+  css: `
+    ^ .foam-u2-DetailView {
+      border: 1px solid #ddd;
+      margin-bottom: 8px;
+    }
+    ^header-row {
+      align-items: center;
+    }
+  `,
+
+  properties: [
+    {
+      name: 'name',
+      documentation: 'This proprerty is used to populate the header for each individual array element'
```

Comment:
typo: "property"
maybe something like "propertyName" would be more descriptive.